docs/organizations/managing-people.md

```diff
@@ -3,43 +3,48 @@
 Members of an organization can see the details of the repositories in that organization and add new repositories to be analyzed by Codacy. Besides this, Codacy only analyzes commits to **private repositories** from contributors who are members of the corresponding organization on Codacy.
 
 !!! important
-    Make sure that you invite or ask your team members to join your organization on Codacy so that Codacy analyzes their commits on private repositories.
+    -   Make sure that you invite or ask your team members to join your organization on Codacy so that Codacy analyzes their commits on private repositories.
 
-    Contributors who aren't part of your Git provider organization can't join your Codacy organization, but you should still add them to Codacy to analyze their commits on private repositories.
+    -   Contributors who aren't part of your Git provider organization can't join your organization on Codacy app, but you should still add them to your Codacy organization to analyze their commits on private repositories.
 
-To list and manage the members of your organization, open your organization **Settings**, page **People**. This page also shows when the organization members last logged in to Codacy:
+    -   In some Enterprise plans, Codacy automatically adds to the organization new people that commit on your private repositories and analyzes their commits. Also, Codacy updates your seats automatically when adding new people.
 
-![People in an organization](images/organization-people.png)
+To list and manage the members of your organization, open your organization **Settings**, page **People**. This page also shows when the organization members last logged in to Codacy app and when they last committed on your repositories:
```

Comment:
Small detail. I believe the more common usage is "commit to a repository":

```suggestion
To list and manage the members of your organization, open your organization **Settings**, page **People**. This page also shows when the organization members last logged in to Codacy app and when they last committed to your repositories:
```

Reply:
I agree with you. I used "commit on" as it was already being used on the page... In fact, a small search tells me that we are using both "commit on" and "commit to" across the documentation. We need to follow up on that later.
I'm updating this one and other occurrences on these two pages for now. 👍